legend-engine-xts-graphQL/legend-engine-xt-graphQL-query/src/main/java/org/finos/legend/engine/query/graphQL/api/execute/GraphQLExecute.java

```diff
@@ -328,6 +331,57 @@ private Response executeGraphQLQuery(String queryClassPath, String mappingPath,
                 }).build();
     }
 
+    public Map<String, ?> computeExtensionsField(String queryClassPath, String mappingPath, String runtimePath, Document document, OperationDefinition query, PureModel pureModel, GraphQLCacheKey graphQLCacheKey, MutableList<CommonProfile> profiles)
```

Comment:
Ideally, plan building for directives should happen along with main plan building - i.e in buildPlanWithParameter method

Reply:
will do

Reply:
done